notebooks/05-operations/01-druid-logs-configuration.ipynb

```diff
@@ -0,0 +1,524 @@
+    "\n",
+    "```bash\n",
+    "cd ~/learn-druid-local/apache-druid/log ; ls\n",
+    "```\n",
+    "\n",
+    "You can see how this results in two sets of files being created:\n",
+    "\n",
+    "* `process name.stdout.log` file - which is the information written by the processes to stdout (i.e., the terminal)\n",
+    "* `process name.log` - file containing various status, error, warning and debug messages"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5258912e-bf23-4acd-9267-a7f4ea3bc6f4",
+   "metadata": {},
+   "source": [
+    "### Log filenames\n",
+    "\n",
+    "The \"FileAppender\" `RollingRandomAccessFile` appender has both `fileName` and `filePattern` properties. The `fileName` property of the \"FileAppender\" sets the name of the log file being written to at the moment, while `filePattern` is applied when the log rolls over.\n",
+    "\n",
+    "Run the following command to change the default filename of the \"FileAppender\" in the log4j configuration file so that all future detailed process logs will have a name suffixed with the hostname.\n",
+    "\n",
+    "```bash\n",
+    "sed -i '' 's/{sys:druid.node.type}.log/{sys:druid.node.type}-${hostName}.log/' \\\n",
+    "  ~/learn-druid-local/apache-druid/conf/druid/auto/_common/log4j2.xml\n",
+    "```\n",
+    "\n",
+    "List the contents of the log folder again to see the changes:\n",
+    "\n",
+    "```bash\n",
+    "ls\n",
+    "```\n",
+    "\n",
+    "Revert the change with the following command:\n",
+    "\n",
+    "```bash\n",
+    "sed -i '' 's/{sys:druid.node.type}-${hostName}.log/{sys:druid.node.type}.log/' \\\n",
+    "  ~/learn-druid-local/apache-druid/conf/druid/auto/_common/log4j2.xml\n",
+    "```\n",
+    "\n",
+    "Since \"FileAppender\" is a [RollingRandomAccessFileAppender](https://logging.apache.org/log4j/log4j-2.4/manual/appenders.html#RollingRandomAccessFileAppender) you can adjust the `TimeBasedTriggeringPolicy` to change when `fileName` log files are rolled over to `filePattern` log files.\n",
+    "\n",
+    "Run the following in your terminal to adjust the `filePattern` to include the hours and minutes.\n",
+    "\n",
+    "```bash\n",
+    "sed -i '' 's/{yyyyMMdd}/{yyyyMMdd-HH:mm}/' \\\n",
+    "  ~/learn-druid-local/apache-druid/conf/druid/auto/_common/log4j2.xml\n",
+    "```\n",
+    "\n",
+    "Run this command to see the new file pattern:\n",
+    "\n",
+    "```bash\n",
+    "grep filePattern ~/learn-druid-local/apache-druid/conf/druid/auto/_common/log4j2.xml\n",
+    "```\n",
+    "\n",
+    "Since the `TimeBasedTriggeringPolicy` is set to 1 by default, a change in the least granular element of the `filePattern` will trigger a rollover. In this case, you will now see a new file being created every minute.\n",
```

Comment:
```suggestion
    "Since the `TimeBasedTriggeringPolicy` is set to 1 by default, a change in the least granular element of the `filePattern` triggers a rollover.\n",
```
We can remove the last sentence since it conveys the same message as the following sentence.